html/js/comp/WifiPage.js

```diff
@@ -20,41 +20,40 @@ export function WifiPage(props) {
     }, []);
 
     function changeWifi() {
-        fetch(props.API + '/api/wifi/set?ssid=' + document.getElementById("ssid").value.trim() + '&pass=' + document.getElementById("pass").value.trim())
-        document.getElementById("ssid").value = '';
-        document.getElementById("pass").value = '';
+        fetch(`${props.API}/api/wifi/set?ssid=${document.getElementById("ssid").value.trim()}&pass=${document.getElementById("pass").value.trim()}`);
+        document.getElementById("ssid").value = "";
+        document.getElementById("pass").value = "";
     }
 
-    var form = <><Form>
-        <p><label for="ssid"><Wifi /> SSID:</label>
-            <input type="text" id="ssid" name="ssid" autocapitalize="none" />
+    const form = <><Form>
+        <p><label htmlFor="ssid"><Wifi /> SSID:</label>
+            <input type="text" id="ssid" name="ssid" autoCapitalize="none" />
         </p>
-        <p><label for="pass"><Lock /> Password:</label>
-            <input type="text" id="pass" name="pass" autocapitalize="none" />
+        <p><label htmlFor="pass"><Lock /> Password:</label>
+            <input type="text" id="pass" name="pass" autoCapitalize="none" />
```

Comment:
This is also a little surprising, but it might be a React-ism I'm not familiar with. As long as the rendered HTML comes out correctly I guess it's fine.

Reply:
These are JSX specific:

Since for is a reserved word in JavaScript, React elements use htmlFor instead.
https://reactjs.org/docs/dom-elements.html

React props use the camelCase syntax.. so autocapitalize needs to be autoCapitalize.
https://stackoverflow.com/questions/40722177/redux-form-field-autocapitalize-attribute